packages/commonwealth/client/scripts/views/Layout.tsx

```diff
@@ -130,6 +131,38 @@ const LayoutComponent = ({
     <ErrorBoundary
       FallbackComponent={({ error }) => <ErrorPage message={error?.message} />}
     >
+      <Helmet>
+        <meta name="title" content="Common" />
+        <meta
+          name="description"
+          content="Discuss, organize, and grow decentralized communities"
+        />
+        <meta name="author" content="" />
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:title" content="Common" />
+        <meta name="twitter:site" content="@hicommonwealth" />
+        <meta
+          name="twitter:description"
+          content="Discuss, organize, and grow decentralized communities"
```

Comment:
This is just a generic fallback version?

We should have metadata here for the page.

Reply:
Not sure why it was originally done this way, but I think updating metadata here for each page would be scope creep for this ticket, which i am fine with if we can delay this PR a bit.

Will create a generic component for metatags tho.